examples/seismic/acoustic/operators.py

```diff
@@ -90,13 +109,20 @@ def ForwardOperator(model, geometry, space_order=4,
 def AdjointOperator(model, geometry, space_order=4,
                     kernel='OT2', **kwargs):
     """
-    Constructor method for the adjoint modelling operator in an acoustic media
+    Constructor an adjoint modelling operator in an acoustic media.
+
+    Parameters
+    ----------
+    model : Model
+        Object containing the physical parameters.
+    geometry : AcquisitionGeometry
+        Geometry object that contains the source (SpareTimeFunction) and
+        receivers (SparseTimeFunction) and their position.
+    space_order : int, optional
+        Space discretization order.
+    kernel :
```

Comment:
we are missing this (@mloubout , @georgebisbas )

Reply:
solved

Reply:
it seems it's still unsolved